docs/en/index.rst

```diff
@@ -237,6 +237,7 @@ This shell can take a few different options:
 
 - ``--config`` (default: default): Name of a queue config to use
 - ``--queue`` (default: default): Name of queue to bind to
+- ``--processor`` (default: ``null``): Name of processor to bind to
```

Comment:
It looks like we use `null` or other defaults, but not sure it makes sense. What is `null` max runtime, for example.

Reply:
I suspect it means use the default defined in enqueue.

Reply:
Thank you, both of you.